lib/index.js

```diff
@@ -107,7 +107,7 @@ api.login = function(req, res, next) {
   })(req, res, next);
 };
 
-api.logout = function(req, res, next) {
+api.logout = function(req, res) {
```

Comment:
Does removing `next` cause any weirdness with express? I think we had that in there before because express used to do some kind of reflection argument length checking or something to determine how to call the middleware. If that's not true anymore (or never was), this is fine. I just don't know.